### DISCUSSION TITLE
attention mask issue

### DISCUSSION THREAD

**Discussion by husai033** [aub-mind/arabert#53] [Q&A] [answered]
I am trying to run the code for Text-classification/Sentiment Analysis with Hyper Parameter tuning available on
https://colab.research.google.com/drive/1P9iQHtUH5KUbTVtp8B4-AopZzEEPE0lw?usp=sharing 
without hyper parameter optimization using the regular training code, but I got the following error when I run trainer.train()
<img width="775" alt="Screen Shot 2021-01-16 at 2 11 01 AM" src="https://user-images.githubusercontent.com/12014572/104805809-b5b73a00-57a0-11eb-8fed-07dba46df2f8.png">

**Comment by WissamAntoun** [comment, marked as answer]
#51
